app/presenters/curation_concerns/generic_work_show_presenter.rb

```diff
@@ -178,6 +178,11 @@ def direct_representative_presenter
       @direct_representative_presenter ||= member_presenters([representative_id]).first
     end
 
+    # Used by _media_player.
+    def viewable_audio_member_presenters
+      viewable_member_presenters.select {|x| x.audio?}
+    end
```

Comment:
maybe a better name for this helper method that makes it clear this is a list of things that will be used with the audio player, displayed in a menu, when you click on them they will play in a player.